runtime/lib/capi/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ target_include_directories(rt_capi PUBLIC .
                                         ${runtime_includes}
                                         ${capi_utils_includes}
                                         ${backend_includes}
+                                        ${CMAKE_DL_LIBS}
```

Comment:
If this is just the `-ldl` flag, I don't think we need it in the include directories right?

Reply:
I removed it while the conversation. https://github.com/PennyLaneAI/catalyst/pull/327/commits/1a2c0d0769b41db7b8bc312df657fb0dd780e221